extensions/incubator/src/main/java/io/opentelemetry/extension/trace/ExtendedTracer.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.extension.trace;
```

Comment:
extension.incubator.trace?

Reply:
No opinion. @bogdandrutu what do you think?

Reply:
Since `moduleName` is `extra["moduleName"] = "io.opentelemetry.extension.incubator"` is it questionable?

Reply:
Do we have a general rule of having the package match the module? I'm fine with that; I just hadn't considered it. :)

Reply:
I thought so at least :) I think so far all our module names are prefixes of the packages in that artifact.

Reply:
done